tests/std/tests/VSO_1062649_overaligned_function/test.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <cstddef>
+#include <cstdint>
+#include <functional>
+
+#ifndef __cpp_aligned_new
+#error This test is only valid when C++17 over-aligned allocation is supported
+#endif
+
+#pragma warning(disable : 4324) // "was padded due to alignment specifier", thanks, Captain

```

Comment:
Much as I agree with the warning not being terribly useful in this test I think such editorializing is somewhat unprofessional.

```suggestion
#pragma warning(disable : 4324) // structure was padded due to alignment specifier

```